web/static/js/tour.js

```
@@ -150,38 +425,191 @@
+        var screen = getCurrentScreen();
+
+        // vNext path: navigate to Dashboard if not already there.
+        if (screen !== null && window.location.pathname.indexOf(bp + '/next') === 0) {
+            if (!isDashboardPage()) {
+                window.markTourPending();
+                window.location.href = bp + '/next/';
+                return;
+            }
+            // Already on the Dashboard -- start immediately.
+            localStorage.removeItem(TOUR_COMPLETED_KEY);
+            var dashSteps = SCREEN_STEPS.dashboard();
```

Biome (2.5.0)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)